src/ostorlab/cli/agent/delete/delete.py

```diff
@@ -5,25 +5,25 @@
 import docker
 from typing import Optional
 from ostorlab.cli import console as cli_console
-from ostorlab.cli.agent import agent
+from ostorlab.cli.agent import agent as agent_cli
 
 console = cli_console.Console()
 
 logger = logging.getLogger(__name__)
 
 
-@agent.command()
-@click.option('--agent-key', '-a', help='Agent Key.', required=True)
+@agent_cli.command()
+@click.argument('agent', required=True)
 @click.option('--agent-version-regex', '-r', help='Agent version matching regular expression.', required=False)
-def delete(agent_key: str, agent_version_regex: Optional[str] = None) -> None:
+def delete(agent: str, agent_version_regex: Optional[str] = None) -> None:
     """CLI command to list installed agents."""
     docker_client = docker.from_env()
     images = docker_client.images.list()
-    agent_container_name = agent_key.replace('/', '_')
+    agent_container_name = agent.replace('/', '_')
     deleted = False
     for im in images:
         for t in im.tags:
-            if t.split(':')[0] == agent_container_name:
+            if t.split(':')[0] in (agent_container_name, f'ostorlab.store/agents/{agent_container_name}'):
```

Comment:
I don't understand the purpose of : (agent_container_name, f'ostorlab.store/agents/{agent_container_name}')

Reply:
The agents when they are installed, produce 2 images, one tagged from the agent name and one with the remote name. Hence we need to delete both.